tests/files/packet_debian11-custom-cni.yml

```diff
@@ -6,4 +6,4 @@ mode: default
 # Kubespray settings
 kube_network_plugin: custom_cni
 custom_cni_manifests:
-  - tests/files/custom_cni/cilium.yaml
+  - "{{ playbook_dir }}/../tests/files/custom_cni/cilium.yaml
```

Comment:
```suggestion
  - "{{ playbook_dir }}/../tests/files/custom_cni/cilium.yaml"
```

Reply:
Indeed, it seems I fixed it up with the test commit. Fixed. Thanks for pointing it out though!